src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -424,6 +426,16 @@ extern "C" {
  */
 typedef VOID (*RtcOnFrame)(UINT64, PFrame);
 
+/*
+ * RtcOnBandwidthEstimation is fired everytime a bandwidth estimation value
+ * is computed. This will be fired for sender or receiver side estimation
+ *
+ * RtcOnBandwidthEstimation is a KVS specific method
+ *
+ */
+typedef VOID (*RtcOnBandwidthEstimation)(UINT64, DOUBLE);
```

Comment:
One more important question. Can the comment contain the link or something to the RFC describing the DOUBLE?

Reply:
In https://tools.ietf.org/html/draft-alvestrand-rmcat-remb-00#section-2.2 it is given as a mantissa + exponent. I chose DOUBLE just because of the return value of `ldexp`